libavcodec/amfenc.c

```diff
@@ -311,8 +312,20 @@ static int amf_init_context(AVCodecContext *avctx)
             if (res == AMF_OK) {
                 av_log(avctx, AV_LOG_VERBOSE, "AMF initialisation succeeded via D3D9.\n");
             } else {
-                av_log(avctx, AV_LOG_ERROR, "AMF initialisation failed via D3D9: error %d.\n", res);
-                return AVERROR(ENOSYS);
+                AMFGuid guid = IID_AMFContext1();
```

Comment:
how we ended up hosting source files of ffmpeg?..

Reply:
These changes should be available in FFmpeg 4.3 but I don't know when it will be released and when jellyfin will follow up on the upstream version.

Reply:
Why don't you include it as a patch, so normal tree stays clean? See here: https://github.com/jellyfin/jellyfin-ffmpeg/tree/jellyfin/debian/patches

Reply:
I have made a patch to solve it : )